container-images/ramalama/Containerfile

```diff
@@ -2,5 +2,5 @@ FROM registry.access.redhat.com/ubi9/ubi:9.5-1742918310
 
 COPY --chmod=755 ../scripts /usr/bin
 
-RUN /usr/bin/build_llama_and_whisper.sh "ramalama" && \
+RUN build_llama_and_whisper.sh "ramalama"
```

Comment:
missing `&& \` here I think (at least it's inconsistent with the `"rocm"` flavor below)

Reply:
Thanks @kpouget nice catch, messed up fixing merge conflicts